google-cloud-logging/lib/google/logging/v2/logging_config_pb.rb

```diff
@@ -16,14 +18,20 @@
     optional :output_version_format, :enum, 6, "google.logging.v2.LogSink.VersionFormat"
     optional :writer_identity, :string, 8
     optional :include_children, :bool, 9
-    optional :start_time, :message, 10, "google.protobuf.Timestamp"
-    optional :end_time, :message, 11, "google.protobuf.Timestamp"
```

Comment:
Removing `LogSink#start_time ` and `LogSink#end_time ` looks like a breaking change.